packages/editor-preview/src/browser/editor-preview-manager.ts

```diff
@@ -1,5 +1,5 @@
 /********************************************************************************
- * Copyright (C) 2018 Google and others.
+ * Copyright (C) 2021 Ericsson and others.
```

Comment:
We should keep the original license (generally we do not update them - they reflect the original author of the file).

Reply:
This is a slightly tricky case - I created whole new files with separate names (and new headers) while I was working on the code, then replaced the old files when I was done, so there is very little lineal connection between the file by this name on `master` and the version here - the diff shows almost only brackets 'carried over'. Since this is a totally breaking change (if you were importing anything from any of these files, it no longer does what you thought), there's also a case for using different file names so that's impossible to pretend that they still do what they used to. I'm not an expert on the ins and outs of the license headers, though, so I defer to the judgment of anyone better informed.

Reply:
Changed to `2018-2021 Google and others`